Doc/library/traceback.rst

```diff
@@ -212,7 +212,7 @@ The module also defines the following classes:
 :class:`TracebackException` objects are created from actual exceptions to
 capture data for later printing in a lightweight fashion.
 
-.. class:: TracebackException(exc_type, exc_value, exc_traceback, *, limit=None, lookup_lines=True, capture_locals=False, compact=False)
+.. class:: TracebackException(exc_type, exc_value, exc_traceback, *, limit=None, lookup_lines=True, capture_locals=False, compact=False, hide_frames=None)
```

Comment:
> Should the line be broken in two, or is one line needed to have the link marker in the right place?

No idea, will try to find out. 

> The parameter name should be single, since the function is applied to one frame at a time.

Agreed.

> A keep function, 'keep_frame', would be used without a wrapper. Is it a burden for user to add 'not' in their code?

I feel like it makes the API less intuitive if you need to actively "keep" a frame rather than actively "hide" it (if you do nothing you hide nothing).

> The new parameter must be added to most or all of the format/print functions that call TracebackException.

In https://github.com/python/cpython/pull/24231  we went the other way  - added .print() to TracebackException so that the module level wrappers handle only the standard use cases and don't need to know about all the configuration options.

Reply:
> > Should the line be broken in two, or is one line needed to have the link marker in the right place?
> 
> No idea, will try to find out.

I tried it - breaking the line does mess up the rendered html.

But while I was there I saw that the from_exception method's doc needs updating too  - will push it shortly.

Reply:
> I feel like it makes the API less intuitive if you need to actively "keep" a frame rather than actively "hide" it (if you do nothing you hide nothing).


How about I call it "frame_filter" and then I think it makes sense - filter needs to return True for something to be included, and no filter means nothing is filtered.